src/components/InputTimeInterval.vue

```diff
@@ -95,9 +111,17 @@ export default {
         this.mode == 'last_duration' ||
         (!this.emptyDaterange && !this.invalidDaterange && !this.daterangeTooLong)
       ) {
+        this.lastUpdate = moment();
         this.$emit('input', this.value);
       }
     },
+    update() {
+      if (this.mode == 'last_duration') {
+        this.mode = ''; // remove cache on v-model
+        this.mode = 'last_duration';
```

Comment:
It's kinda wired that the button won't work without line 120. 😣
I guess the cause is that the computed property `value` (line 85) would be cached by Vue. You may refer to [this example from official documentation](https://vuejs.org/guide/essentials/computed.html#computed-caching-vs-methods). 
The documentation suggests to use methods instead, which is not avaliable for our situation, since it is difficult to call the methods from parent components.
I also tried to use a function (like `()=>[moment().subtract(this.duration, 'seconds'), moment()]`) as the return value of the getter, but this will disable the watchers in Timeline and make things more complex.
So I figured out this simple-yet-strange solution: I can change the value of a reactive dependency (`this.mode`) to something else in order to invoke a update on the cached value. And to remain consistency it is changed it back immediately. After `this.update()`, the cached values are updated so the watchers in the parent components would be invoked, resulting a update in the timeline.

Reply:
Ah, I get it now. Thanks for the explanation!

I could probably figure out another way to get around it (make the `value` computed depend on the `lastUpdated` reactive?), but this seems good enough.